com.unity.render-pipelines.high-definition/Editor/Lighting/Reflection/HDScreenSpaceReflectionEditor.cs

```diff
@@ -80,6 +82,7 @@ public override void OnEnable()
         static public readonly GUIContent k_RayTracingText = EditorGUIUtility.TrTextContent("Ray Tracing (Preview)", "Enable ray traced reflections.");
         static public readonly GUIContent k_ReflectSkyText = EditorGUIUtility.TrTextContent("Reflect Sky", "When enabled, SSR handles sky reflection.");
         static public readonly GUIContent k_LayerMaskText = EditorGUIUtility.TrTextContent("Layer Mask", "Layer mask used to include the objects for screen space reflection.");
+        static public readonly GUIContent k_LodBiasText = EditorGUIUtility.TrTextContent("Lod Bias", "Defines the LOD Bias for sampling all the textures. A higher value will increase peformance and make denoising easier but may reduce visual fidelity.");
```

Comment:
The LOD Bias for sampling all the textures. A higher value increases performance and makes denoising easier, but it might reduce visual fidelity.